<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>322</width>
    <height>461</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>322</width>
    <height>461</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kalkulator</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #2E2E2E;
}

QPlainTextEdit {
    background-color: #1E1E1E;
    color: #FFFFFF;
    border: 1px solid #444444;
    border-radius: 5px;
    padding: 5px;
	font-size: 24px;
	text-align: right;
}</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>459</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0" columnstretch="0,0,0,0">
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4E4E4E;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #5E5E5E;
}

QPushButton:pressed {
    background-color: #3E3E3E;
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="buttonMultiply">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4E4E4E;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #5E5E5E;
}

QPushButton:pressed {
    background-color: #3E3E3E;
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4E4E4E;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #5E5E5E;
}

QPushButton:pressed {
    background-color: #3E3E3E;
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="buttonSin">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="buttonSubtract">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4E4E4E;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #5E5E5E;
}

QPushButton:pressed {
    background-color: #3E3E3E;
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4E4E4E;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #5E5E5E;
}

QPushButton:pressed {
    background-color: #3E3E3E;
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QPlainTextEdit" name="display">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4E4E4E;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #5E5E5E;
}

QPushButton:pressed {
    background-color: #3E3E3E;
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_0">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4E4E4E;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #5E5E5E;
}

QPushButton:pressed {
    background-color: #3E3E3E;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="buttonAdd">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBoxRadians">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color: #FFFFFF;
    spacing: 5px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
    border: 1px solid #FFFFFF;
    background-color: #2E2E2E;
}

QCheckBox::indicator:unchecked:hover {
    border: 1px solid #FFA500;
    background-color: #3E3E3E;
}

QCheckBox::indicator:checked {
    border: 1px solid #FFA500;
    background-color: #FF8C00;
}

QCheckBox::indicator:checked:hover {
    border: 1px solid #FFA500;
    background-color: #FFA500;
}</string>
       </property>
       <property name="text">
        <string>Rad</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4E4E4E;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #5E5E5E;
}

QPushButton:pressed {
    background-color: #3E3E3E;
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="buttonPoint">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4E4E4E;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #5E5E5E;
}

QPushButton:pressed {
    background-color: #3E3E3E;
}</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="buttonDivide">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4E4E4E;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #5E5E5E;
}

QPushButton:pressed {
    background-color: #3E3E3E;
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="buttonCos">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="buttonEquals">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4E4E4E;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #5E5E5E;
}

QPushButton:pressed {
    background-color: #3E3E3E;
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="buttonClear">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="buttonSquare">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>x^2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="buttonClearEntry">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>CE</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="buttonChangeSign">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="buttonSqrt">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>sqrt</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="buttonLn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="buttonReciprocal">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF8C00;
    color: #FFFFFF;
    border: none;
    border-radius: 5px;
    padding: 10px;
}

QPushButton:hover {
    background-color: #FFA500;
}

QPushButton:pressed {
    background-color: #FF7F00;
}</string>
       </property>
       <property name="text">
        <string>1/x</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBoxAdvancedFunctions">
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color: #FFFFFF;
    spacing: 5px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
    border: 1px solid #FFFFFF;
    background-color: #2E2E2E;
}

QCheckBox::indicator:unchecked:hover {
    border: 1px solid #FFA500;
    background-color: #3E3E3E;
}

QCheckBox::indicator:checked {
    border: 1px solid #FFA500;
    background-color: #FF8C00;
}

QCheckBox::indicator:checked:hover {
    border: 1px solid #FFA500;
    background-color: #FFA500;
}</string>
       </property>
       <property name="text">
        <string>More</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
